docs/conf/na/2017/unconference.rst

```diff
@@ -46,7 +46,7 @@ Schedule a Session
 ------------------
 
 Attendees that wish to lead an unconference session have several options
-for signing up. We will have a board with all of the sessions for the
+for signing up. We will have a 'board <https://docs.google.com/spreadsheets/d/1Im3Ce1-JubF33Ydg4XPCnafZbkVMKXDtWE9uoa_SskI/edit?usp=sharing>`__ with all of the sessions for the
```

Comment:
The link doesn't seem to resolve in rendering. The quotes need to be slanted (``) not straight ('') in the beginning and the end. Also, need to remove one underscode, but that's more for cleanness than function :)